OutOfSchool/OutOfSchool.ExternalFileStore.Impl/NotificationImplementations/MinioNotificationBackgroundService.cs

```diff
@@ -0,0 +1,120 @@
+﻿using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using OutOfSchool.ExternalFileStore.Models;
+using OutOfSchool.ExternalFileStore.NotificationInterfaces;
+
+namespace OutOfSchool.ExternalFileStore.NotificationImplementations;
```

Comment:
same comment for namespace. Make the classes internal and handle DI registration in extension method in implementation project